src/web/lite.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+    <meta name="description" content="">
+    <meta name="keywords" content="">
+    <link rel="icon" type="image/png" href="/img/favicon-32x32.png" sizes="32x32" />
```

Comment:
Good point I think I'll remove those lines for now ad I'm not sure if this is going to be used at all?